src/backend/people/settings.py

```diff
@@ -291,6 +291,7 @@ class Base(Configuration):
 
     # Session
     SESSION_ENGINE = "django.contrib.sessions.backends.cache"
+    SESSION_CACHE_ALIAS = "default"
```

Comment:
I don't think you should use the default cache for this because people will use the default cache for something else. Call it "session"?

Reply:
fixed